src/Main.java

```diff
@@ -24,6 +26,7 @@ public static void main(String[] args) {
                 try {
                     placeHolderUserInput = userInput.nextInt();
 
+                    //checking if input is between 0 and 10
                     if (0 <= (int) placeHolderUserInput && 10 > placeHolderUserInput) {
```

Comment:
Had liever gezien (stond volgens mij ook in de opdracht maar kan het 1,2,3, even niet terug vinden) dat je .length gebruikt. Nu is er ietwat meer werk vereist om het programma uit te breiden naar meer antwoordmogelijkheden.